packages/api/src/schema/schema.yaml

```diff
@@ -77,6 +77,9 @@ components:
           description:
             If true, returning non 2xx value from webhook will prevent stream
```

Comment:
Unrelated: Should we remove this `blocking` property now that we don't support it?